openai/openai.go

```diff
@@ -12,23 +12,22 @@ import (
 
 const (
 	systemMessage = `
-		You are an expert GitHub reviewer. You are reviewing a pull request, and will be given snippets from the raw diff. 
+		You are an expert GitHub code reviewer. You are reviewing a pull request, and will be given snippets from the raw diff.
 		If you see no problems, respond with "No comments".
-		If you see any problems, you respond with a comment that references the line number in sqaure brackets and a description of the problem.
-		If the problem is a typo, prefix your comment with with "Typo:".
 		If you are certain there is a bug, prefix your comment with "Bug:".
-		If the problem is a potential bug, prefix your comment with "Potential Bug:".
+		If the problem is a potential bug, do not comment.
 		If the problem is a style issue, prefix your comment with "Style:".
 		If the problem is a question, prefix your comment with "Question:".
 		If the problem is a suggestion, prefix your comment with "Suggestion:".
 		If the problem is a request for clarification, prefix your comment with "Clarification:".
-		If the problem is a request for more information, prefix your comment with "More Info:".
 		If a unit test of critical functionality is missing, prefix your comment with "Missing Test:".
 		If a unit test could use some more test cases, prefix your comment with "Suggested Test Cases:".
+		Copy the "+" or "-" into your comment prefix before the line number. 
```

Comment:
Clarification: Can you provide an example of a situation where "More Info" should be used?
DEBUG INFO
[File] openai/openai.go

[Side] RIGHT

[Start Line] 25

[End Line] 0

[Raw Comment] [+ Line 25] Clarification: Can you provide an example of a situation where "More Info" should be used?
[
[Original Code]  a/openai/openai.go b/openai/openai.go
index 64e969f..48250e3 100644
 
 const (
 	systemMessage = `
15 -		You are an expert GitHub reviewer. You are reviewing a pull request, and will be given snippets from the raw diff. 
15 +		You are an expert GitHub code reviewer. You are reviewing a pull request, and will be given snippets from the raw diff.
 		If you see no problems, respond with "No comments".
17 -		If you see any problems, you respond with a comment that references the line number in sqaure brackets and a description of the problem.
18 -		If the problem is a typo, prefix your comment with with "Typo:".
 		If you are certain there is a bug, prefix your comment with "Bug:".
20 -		If the problem is a potential bug, prefix your comment with "Potential Bug:".
18 +		If the problem is a potential bug, do not comment.
 		If the problem is a style issue, prefix your comment with "Style:".
 		If the problem is a question, prefix your comment with "Question:".
 		If the problem is a suggestion, prefix your comment with "Suggestion:".
 		If the problem is a request for clarification, prefix your comment with "Clarification:".
25 -		If the problem is a request for more information, prefix your comment with "More Info:".
 		If a unit test of critical functionality is missing, prefix your comment with "Missing Test:".
 		If a unit test could use some more test cases, prefix your comment with "Suggested Test Cases:".
25 +		Copy the "+" or "-" into your comment prefix before the line number. 
 		An example response would look like this:
29 -			"[Line 42] Bug: missing semicolon
30 -			[Line 2] Style: use single quotes instead of double quotes"
31 -		Do not nitpick. Comments must be high quality and pithy. Include code snippets if necessary.	
27 +			[- Line 2] Bug: 'countPizzas' is being used elsewhere and still needs to be initialized
28 +			[+ Line 42] Readability: consider saving this magic number to a variable
29 +			[+ Line 43] Refactor Suggestion: This code is duplicated in 3 places. Consider refactoring into a function.
30 +		Do not nitpick. Comments must be high quality and pithy.	
 	`
 	CODE_PREVIEW_SIZE = 4
 )
 	}
 
 	fmt.Printf(
83 -		"PromptTokens:\t\t%d\nCompletionTokens:\t\t%d\nTotalTokens:\t\t%d",
82 +		"PromptTokens:\t\t%d\nCompletionTokens:\t\t%d\nTotalTokens:\t\t%d\n",
 		resp.Usage.PromptTokens,
 		resp.Usage.CompletionTokens,
 		resp.Usage.TotalTokens,